extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -89,6 +89,16 @@ define('ext.wikia.adEngine.adContext', [
 			context.providers.turtle = true;
 		}
 
+		// Monetization Service
+		if (context.forceProviders.monetizationService) {
+			context.providers.monetizationService = true;
```

Comment:
I don't think this is needed (compare to the condition ``if (context.providers.monetizationService || context.forceProviders.monetizationService) {`` from above)